src/api/TaskAssignments.js

```diff
@@ -10,9 +10,28 @@ import {
     ERROR_CODE_CREATE_TASK_ASSIGNMENT,
     ERROR_CODE_UPDATE_TASK_ASSIGNMENT,
     ERROR_CODE_DELETE_TASK_ASSIGNMENT,
+    HTTP_STATUS_CODE_RATE_LIMIT,
+    HTTP_STATUS_CODE_INTERNAL_SERVER_ERROR,
+    HTTP_STATUS_CODE_NOT_IMPLEMENTED,
+    HTTP_STATUS_CODE_BAD_GATEWAY,
+    HTTP_STATUS_CODE_SERVICE_UNAVAILABLE,
+    HTTP_STATUS_CODE_GATEWAY_TIMEOUT,
 } from '../constants';
 
+const RETRYABLE = [
+    HTTP_STATUS_CODE_RATE_LIMIT,
+    HTTP_STATUS_CODE_INTERNAL_SERVER_ERROR,
+    HTTP_STATUS_CODE_NOT_IMPLEMENTED,
```

Comment:
Is it possible that retrying not_implemented could succeed?

Reply:
Looking at other retry plugins, they retry all 5xx codes:

https://github.com/JustinBeckwith/retry-axios/blob/master/src/index.ts#L110-L120

https://github.com/softonic/axios-retry/blob/master/es/index.js#L28

Reply:
It looks like they also retry put/delete. Do the task service endpoints throw 501 errors that we want to retry?

Reply:
One of them retries PUT and DELETE, the other doesn't. Technically those are supposed to be idempotent but 1) we aren't seeing many errors on those methods. 99%+ are GET. 2) Due to the side-effects on task status when task_collaborators are updated, I am not sure we want to retry PUT and DELETE for this particular case.

Reply:
I'll check with backend about what they recommend

Reply:
We are going to go with what's here. Not PUT and DELETE.